spec/USERS.md

```diff
@@ -0,0 +1,70 @@
+# Test Users
```

Comment:
I'm not sure if there are any conventions around including more than one markdown file in a folder. I named this file USERS.md, but I'm happy to change it if there's another naming scheme you prefer.

Reply:
I think it's fine for now, we could revisit in the future if more files are added.